test/Microsoft.Framework.Runtime.Tests/ProjectFilesCollectionFacts.cs

```diff
@@ -59,6 +59,30 @@ public void FilePatternsWorkForArraysAreSet()
             Assert.Equal(new[] { "a.cs", @"foo.js" }, target.ResourcesPatterns);
         }
 
+
+        [Fact]
+        public void FilePatternsDoNotIncludeComments()
+        {
+            var rawProject = Deserialize(@"
+{
+    ""code"": [""*.cs"", /* FOO */ ""../*.cs""],
+    ""exclude"": [""buggy/*.*"", /* FOO */],
```

Comment:
Thanks for the feedback!  I can address the issues you've brought up.  Does the general concept look good, though or should I take a different approach?

Sent from my Windows Phone

-----Original Message-----
From: "Eilon Lipton" <notifications@github.com>
Sent: ‎2/‎19/‎2015 3:37 PM
To: "aspnet/XRE" <XRE@noreply.github.com>
Cc: "Andrew Benz" <andorbal@gmail.com>
Subject: Re: [XRE] Exclude comments when getting JToken values as an array(#1247)

In test/Microsoft.Framework.Runtime.Tests/ProjectFilesCollectionFacts.cs:
> @@ -59,6 +59,30 @@ public void FilePatternsWorkForArraysAreSet()
>              Assert.Equal(new[] { "a.cs", @"foo.js" }, target.ResourcesPatterns);
>          }
>  
> +
> +        [Fact]
> +        public void FilePatternsDoNotIncludeComments()
> +        {
> +            var rawProject = Deserialize(@"
> +{
> +    ""code"": [""*.cs"", /* FOO */ ""../*.cs""],
> +    ""exclude"": [""buggy/*.*"", /* FOO */],
BTW there are a couple of these that aren't really valid - there's an undefined last element in the array because there's a trailing comma. There's another example below in shared.
—
Reply to this email directly or view it on GitHub.